test/test_torch.py

```diff
@@ -1391,7 +1398,12 @@ def _test_broadcast(self, cast):
                 small2 = cast(torch.randn(*dims_small2).float())
                 small2_expanded = small2.expand(*dims_full)
 
-            if hasattr(large_expanded, fn):
+            if small.is_cuda and fn in ['map', 'map2']:
+                # map and map2 are not implementd on CUDA tensors
+                continue
+
+            # TODO: fix masked_scatter and masked_fill broadcasting
```

Comment:
what exactly is the problem?

Reply:
They use in-place broadcasting rules instead of out-of-place. This is because they're implemented in terms of the out-of-place functions.

Reply:
It's not a breaking change. These didn't exist on Tensor.

Reply:
Good point, I made this a "new issue".